org.eclipse.january.examples/src/org/eclipse/january/examples/dataset/BasicExample.java

```diff
@@ -63,6 +64,15 @@ public static void main(String[] args) {
 		Dataset slice = dataset.getSlice(new Slice(0, 2), new Slice(1, 3));
 		System.out.println("slice of dataset: \n" + slice.toString(true));
 
+
+		// Random dataset
+		Dataset a = DatasetFactory.createRange(15).reshape(3, 5);
```

Comment:
This is expected to be integers as in the original example. Try createRange(IntegerDataset.class, 15)